pkg/testutil/testutil.go

```diff
@@ -0,0 +1,285 @@
+package testutil
```

Comment:
I think all of this is useful for only the database. I think this would also be fine being just in the database package.

Also, copyright header